OpenSim/Simulation/Model/ConditionalPathPoint.h

```diff
@@ -50,6 +50,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ConditionalPathPoint, PathPoint);
         "The minimum and maximum values that the coordinate can range between, "
         "for which the PathPoint is active. Angular coordinates in radians.");
 
+//==============================================================================
+// CONNECTORS
+//==============================================================================
+    OpenSim_DECLARE_CONNECTOR(coordinate, Coordinate,
+        "The coordinate whose value this path point depends on.");
```

Comment:
Minor, but it might be helpful for the comment to relate to the Coordinate to the condition used to determine if the path point is active.